libnavui-voice/src/main/java/com/mapbox/navigation/ui/views/soundbutton/SoundButton.kt

```diff
@@ -0,0 +1,268 @@
+package com.mapbox.navigation.ui.views.soundbutton
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.util.AttributeSet
+import android.view.animation.AccelerateInterpolator
+import android.view.animation.AlphaAnimation
+import android.view.animation.Animation
+import android.view.animation.AnimationSet
+import android.view.animation.DecelerateInterpolator
+import android.widget.TextView
+import androidx.annotation.StyleRes
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import androidx.core.graphics.drawable.DrawableCompat
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import com.mapbox.navigation.ui.base.internal.utils.MultiOnClickListener
+import com.mapbox.navigation.ui.voice.R
+
+/**
+ * Sound Button view.
+ */
+class SoundButton : ConstraintLayout {
+
+    private lateinit var soundFab: FloatingActionButton
+    private lateinit var soundChipText: TextView
+    private var fadeInSlowOut: AnimationSet? = null
+    private var multiOnClickListener: MultiOnClickListener? = MultiOnClickListener()
+
+    private var primaryColor = 0
+    private var secondaryColor = 0
+
+    /**
+     * *true* if it's muted, otherwise *false*
+     */
+    var isMuted = false
+        private set
+
+    private companion object {
+        private const val ALPHA_VALUE_ZERO = 0f
+        private const val ALPHA_VALUE_ONE = 1f
+        private const val ANIMATION_DURATION_THREE_HUNDRED_MILLIS: Long = 300
+        private const val ANIMATION_DURATION_ONE_THOUSAND_MILLIS: Long = 1000
```

Comment:
```suggestion
        private const val VALUE_ZERO = 0f
        private const val VALUE_ONE = 1f
        private const val THREE_HUNDRED_MILLIS: Long = 300
        private const val ONE_THOUSAND_MILLIS: Long = 1000
```

Reply:
I think we should keep names to be obvious what they use for